INSTALLATION.md

```diff
@@ -0,0 +1,97 @@
+# Installing the tools
+In this file you'll find instructions on how to install the tools we'll use during the course.
+
+All of these tools are available for Linux, Mac and Windows users. We'll need the tools to write and compile our Rust code, and allow for remote mentoring. *Important: these instructions are to be followed at home, before the start of the first tutorial. If you have any problems with installation, contact the lecturers! We won't be adressing installation problems during the first tutorial.*
+
+## Rust and Cargo
+First we'll need `rustc`, the standard Rust compiler. `rustc` is generally not invoked directly, but through `cargo`, the Rust package manager. `rustup` takes care of installing `rustc` and `cargo`.
+
+This part is easy: go to <https://rustup.rs> and follow the instructions. Please make sure you're installing the default toolchain. Once done, run
+
+```bash
+rustc -V && cargo -V
+```
+
+The output should be something like this:
+
+```bash
+rustc 1.63.0 (4b91a6ea7 2022-08-08)
+cargo 1.63.0 (fd9c4297c 2022-07-01)
+```
+
+Using Rustup, you can install Rust toolchains and components. More info: 
+- <https://rust-lang.github.io/rustup>
+- <https://doc.rust-lang.org/cargo>
+
+## Rustfmt
+To avoid discussions, Rust provides its own formatting tool, `rustfmt`. You can install it using Rustup with the following command:
```

Comment:
`rustup install stable` (the command to install cargo) nowadays installs both clippy and rustfmt by default.
![image](https://user-images.githubusercontent.com/22172241/197528065-e412e8c1-126c-473b-b22e-65f880a9e8f2.png)

Reply:
Oh wow, how nice!